src/main/java/frc/robot/Robot.java

```diff
@@ -7,6 +7,10 @@
 
 package frc.robot;
 
+import java.io.PrintWriter;
```

Comment:
Any reason to move these? Could be a setting in your IDE that is doing this?

Reply:
No, I'm not sure what caused this. I'm not sure if the order makes a difference, but I'll put them back in the original order anyway.

Reply:
This is default behavior in IntelliJ, it organizes imports alphabetically.